app/screens/LocationDetails.js

```diff
@@ -137,6 +137,7 @@ class LocationDetails extends Component {
             const machineDetails = this.props.machines.machines.find(m => m.id === machine.machine_id)
             return {...machineDetails, ...machine}
         }))
+        const { icon: locationIcon, library: iconLibrary } = this.props.locations.locationTypes.find(type => type.id === location.location_type_id)
```

Comment:
This is causing problems for locations without a location_type_id. I will fix to safeguard agains the app crashing. An example location is the Portland Fuddruckers https://pinballmap.com/api/v1/locations/1028.json. I think we should be safeguarding either way, but this could be an opportunity to audit locations without a location type.

Reply:
There are ~1,000 locations that don’t have Type, so it’s definitely something we need to account for. I tried to check on this, but I didn’t have any near me! I should have looked harder.

Reply:
No worries, this is exactly why it's good to have a few people working on this thing!